src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Dont use plural form for enum
```suggestion
public enum Color {

```